### PR TITLE
fswebcam as an alternative command to capture webcam images

### PR DESCRIPTION
**Pull request by pbabinca**
Some cameras (e.g. Genius iSlim 1322AF) takes couple of seconds/frames until they settle on autofocused images. Default command "streamer" doesn't have an option to wait.

This change introduces alternative command - "fswebcam" - for those cameras that are explicitly listed as keys of associative array fswebcam_args. Value of the array is list of arguments to append to default fswebcam args.

### CODE REVIEW

File: config.php [hunk 1]
```diff
@@ -20,6 +20,12 @@
 // resolution of images
 $img_res_preview = "320x240";
 $img_res_full = "1280x720";
+// arguments of fwswebcam indexed by device file names of cameras to apply them to
```
Comment (yarda) [inline config.php]: Could you add comment that without args the streamer command is used?

**Comment by yarda** [comment]
The camera IDs can reorder maybe better to match on VID:PID which should be more robust.

I know we have the blacklist feature which has the same problem, but it was introduced like a hack to blacklist defective laptop internal camera, which is hardwired and usually don't change ID. VID:PID matching would be also better here.

**Comment by yarda** [comment]
I would match them either by the bus_info, taking into account the camera is connected to the same USB port (which IMHO shouldn't reorder) or by the VID:PID which is probably the most robust way. For the VID:PID I think you could decode it from the bus_info from the /sys, or query it through the udevadm, e.g.:
```
VID=`udevadm info /dev/video0 | grep ID_USB_VENDOR_ID= | cut -d= -f 2`
PID=`udevadm info /dev/video0 | grep ID_USB_MODEL_ID= | cut -d= -f 2`
```

**Comment by pbabinca** [comment]
Cameras can be now matched by attributes obtained from `udevadm` tool. As a result removal of non-capturing devices is more stable and it is even possible to remove internal camera by its name.

I understand this is not simple change and I can update 3rd line of `printers.php` and `camera.php` if necessary.

**Comment by yarda** [comment]
Thanks, LGTM, sorry for the delay.

Please next time squash commits into one.

Maybe you could push it further and be more generic about the used streamer, i.e. not hardcode the fswebcam / streamer. E.g. you could define the streamer commands in the config file and then just select the individual streamer by the match_rules. You could also use `additional_args` instead of the `fswebcam_args` to customize the streamer commands by the match_rules. But merging it for now, because it's definitely improvement.

**Pull request merged**